ansible_bender/cli.py

```diff
@@ -310,6 +311,24 @@ def _build(self):
     def _build_inside_openshift(self):
         build_inside_openshift(self.app)
 
+    def fancy_time(build_time):
+        """ 
+        returns build time in human readable form 
+
+        :param build_time: total time taken for the build
```

Comment:
`build_time = b.build_finished_time - b.build_start_time`

it's actually time delta

Reply:
> Sadly, it doesn't work:
> 
> ```
> Traceback (most recent call last):
>   File "/home/tt/.local/bin//ansible-bender", line 11, in <module>
>     load_entry_point('ansible-bender', 'console_scripts', 'ansible-bender')()
>   File "/home/tt/g/TomasTomecek/ansible-bender/ansible_bender/cli.py", line 414, in main
>     sys.exit(CLI().run())
>   File "/home/tt/g/TomasTomecek/ansible-bender/ansible_bender/cli.py", line 384, in run
>     self._list_builds()
>   File "/home/tt/g/TomasTomecek/ansible-bender/ansible_bender/cli.py", line 340, in _list_builds
>     build_time = fancy_time(build_time)
> NameError: name 'fancy_time' is not defined
> ```
> 
> Once you move the function to utils, please write a test case for it.

Added a unit test 😄

Reply:
> `build_time = b.build_finished_time - b.build_start_time`
> 
> it's actually time delta

@TomasTomecek Not sure what should I do here, please help me.

Reply:
I have worked out using timedelta 😄